tests/sample_module.py

```diff
@@ -19,7 +19,7 @@ class SomeClass:
     other_class_attribute = OtherClass
     typedattr: str = "bruh"
 
-    def method(self):
+    def method(self, *args, **kwargs):
```

Comment:
If these changes are no longer needed then they can be removed

Reply:
Shall I remove it and commit again?